### PR TITLE
Add PassiveArbitrator

### PR DESCRIPTION
**Pull request by bbenligiray**
An example use case of https://github.com/api3dao/claims-manager/pull/86

If only the DAO is the manager/admin, this arbitrator can be authorized to operate with no quota. The arbitrator can only create disputes and not resolve them, meaning that only ClaimsManager admins/manager are able to resolve such disputes. This would be useful if Kleros fails to scale with the amounts we cover.

### CODE REVIEW

File: contracts/arbitrators/PassiveArbitrator.sol [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ) external override {
+        // claimsManager.createDispute() will validate the arguments so we don't need to
+        require(msg.sender == claimant, "Sender not claimant");
+        claimsManager.createDispute(
+            policyHash,
+            claimant,
+            beneficiary,
+            claimAmountInUsd,
+            evidence
+        );
```
Comment (bbenligiray) [inline contracts/arbitrators/PassiveArbitrator.sol:30]: I figure there is no need to emit an event here 🤔

**Comment by bbenligiray** [comment]
Currently the frontend only uses the Kleros arbitrator. This is a DAO-governed alternative (given that the DAO is the manager/admin) if we ever need one. I added it so that it's audited and ready to go just in case.

**Comment by acenolaza** [comment]
What confuses me is that we require that `msg.sender` is the `claimant` in this arbitrator contract but then CaimsManager will check then check that `msg.sender` has the `arbitratorRole` so does this mean that we could ever give this role to the `claimant`?

**Comment by bbenligiray** [comment]
`msg.sender` is updated during external calls so the `msg.sender` ClaimsManager will see will be PassiveArbitrator, not the claimant that calls PassiveArbitrator

**Comment by acenolaza** [comment]
Openzeppelin's Multicall calls [functionDelegateCall](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Multicall.sol#L20) from the Address.sol contract which uses [delegatecall](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Address.sol#L185) under the hood. Doesn't that mean that `msg.sender` is going to be kept unaltered until it reaches the target contract?

**Comment by bbenligiray** [comment]
Openzeppelin's Multicall (unlike MakerDAO's Multicall) is designed to do internal calls (because it delegate-calls `address(this)`) which is why its `msg.sender` will not change (only external calls change `msg.sender`). I assume you're using Multicall just as an example because I can't be sure how it's relevant here.

**Comment by acenolaza** [comment]
Yes, I'm using Openzeppelin's multicall as a use case example because PassiveArbitrator inherits from it.

**Comment by bbenligiray** [comment]
`claimant` calls `PassiveArbitrator.multicall()`, which calls `PassiveArbitrator.createDispute()` multiple times (`msg.sender` is `claimant`), which calls `ClaimsManager.createDispute()` (`msg.sender` is PassiveArbitrator). Don't know if this makes sense.
Also, I suppose it's kinda pointless for PassiveArbitrator to inherit Multicall because even if you'd potentially want to create disputes in batch, we won't design/implement the interface for that.

**Comment by acenolaza** [comment]
Clicked on merge by accident. I meant to approve the PR 🤦🏻

**Pull request merged**